gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerKotlinSupportPlugin.kt

```diff
@@ -25,6 +26,23 @@ class ComposeCompilerKotlinSupportPlugin : KotlinCompilerPluginSupportPlugin {
                     ComposeCompilerCompatibility.compilerVersionFor(target.getKotlinPluginVersion())
             }
         }
+        warnAboutJetpackComposeCompilerUsageForNonJvm(target)
+    }
+
+    @Suppress("NON_EXHAUSTIVE_WHEN")
+    private fun warnAboutJetpackComposeCompilerUsageForNonJvm(target: Project) {
+        val isUsingJetpackComposeCompilerPlugin =
+            composeCompilerArtifactProvider.compilerArtifact.groupId.startsWith("androidx.compose.compiler")
```

Comment:
Maybe we should check `!composeCompilerArtifactProvider.compilerArtifact.groupId.startsWith("org.jetbrains.compiler")`

Reply:
good idea. done